cellrank/tl/_lineages.py

```diff
@@ -97,17 +80,16 @@ def lineage_drivers(
     adata: AnnData,
     backward: bool = False,
     lineages: Optional[Union[Sequence, str]] = None,
-    method: str = TestMethod.FISCHER.s,
+    method: str = TestMethod.FISCHER,
```

Comment:
Is any string admissible? No, right? Should be either

```suggestion
    method: TestMethod = TestMethod.FISCHER,
```

if this works or something along the lines of

```suggestion
method: Literal["fisher", "perm_test"] = TestMethod.FISCHER,
```

Reply:
Done (the 2nd variant).